File: php/Dadu.php
<?php
	class Dadu{
		
		public $max_Value;
		
		public function __construct($maxValueInput){
			$this->max_Value=$maxValueInput;
		}
		
		public function get_Max_Value(){
			return $this->max_Value;
		}
		
		public function set_MaxValue($xInput){
			$this->max_Value=$xInput;
		}
		
		public function roll(){
			return $x=rand(1,$this->get_Max_Value());
		}
		
	}
?>

File: old/Dadu.java

import java.util.Random;

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
/**
 *
 * @author i14032
 */
public class Dadu {

    public int maxValue;

    public Dadu(int maxV) {
        this.maxValue = maxV;
    }

    public int getMaxValue() {
        return maxValue;
    }

    public void setMaxValue(int maxValue) {
        this.maxValue = maxValue;
    }

    public int roll() {
        Random rd = new Random();
        return rd.nextInt(this.getMaxValue()) + 1;
    }

}
